### PR TITLE
DESCW-472 Site builder script.

### PR DESCRIPTION
**Pull request by ShawnTurple**

### CODE REVIEW

File: openshift/templates/site-builder.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    echo >&2 "Project:   ${OC_DEPLOY_PROJECT}"
+    echo >&2 "App Name:  ${APP_DOMAIN}"
+    echo >&2 "Site Name: ${SITE_NAME}"
+    echo >&2 "Pool Name: ${POOL_NAME}"
+    
+    whoAmI="$(oc whoami 2> /dev/null)"
+    # This means i am logged in.
+    if [ ! -z "${whoAmI}" ]; then
+        templates="$(dirname $0)";
+        #apply=""
```
Comment (Baelx) [inline openshift/templates/site-builder.sh:14]: should this comment be here?

File: openshift/templates/site-builder.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        apply=" | oc apply -f -"
```
Comment (Baelx) [inline openshift/templates/site-builder.sh:15]: Does this variable get used anywhere?

File: openshift/templates/site-builder.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        echo "Setting up volumes"
+        oc process -p ENV_NAME=${OC_DEPLOY_PROJECT} -p POOL_NAME=${POOL_NAME} -f ${templates}/backups/volume-backup.yaml | oc apply -f -
+        oc process -p ENV_NAME=${OC_DEPLOY_PROJECT} -p POOL_NAME=${POOL_NAME} -f ${templates}/backups/volume-verification.yaml | oc apply -f -
+        echo "Setting up cron conainer"
+        oc process -p ENV_NAME=${OC_DEPLOY_PROJECT} -p SITE_NAME=${SITE_NAME} -p POOL_NAME=${POOL_NAME} -f ${templates}/backups/cron.yaml | oc apply -f -
+    else
+        printf >&2 "$(oc whoami)"
+    fi
+else
+    echo >&2 'Require variables to be set, run: export OC_DEPLOY_PROJECT="test" APP_DOMAIN="des-test.apps.silver.devops.gov.bc.ca" SITE_NAME="des" POOL_NAME="pool-name"'
```
Comment (Baelx) [inline openshift/templates/site-builder.sh:56]: Will every user who uses this wordpress platform want to pass in "DES"-related environment variables? Should this be made more general?

**Review by Baelx — APPROVED** [review]

File: openshift/templates/site-builder.sh (hunk 3 above)
Comment (ShawnTurple) [reply openshift/templates/site-builder.sh:56]: not sure of your question here... this is a tool for us to use to create sites, not for anyone to use.

File: openshift/templates/site-builder.sh (hunk 2 above)
Comment (ShawnTurple) [reply openshift/templates/site-builder.sh]: actual it doesn't it had a purpose, but didn't work, removed it.. 
Thanks.

**Pull request merged**